lib/models/blueprint.js

```diff
@@ -850,7 +850,9 @@ Blueprint.prototype._locals = function(options) {
   var sanitizedModuleName = moduleName.replace(/\//g, '-');
 
   return new Promise(function(resolve) {
-    resolve(this.locals(options));
+    return resolve(this._processPrompts(options));
+  }.bind(this)).then(function (prompts) {
```

Comment:
I was actually surprised to see that an exception inside the `new Promise()` callback will reject the promise instead of propagating to the calling context. IMHO it looks a bit cleaner that way, but I'm fine with going either way.